deployments/Dockerfiles/localnode/app.py

```diff
@@ -41,8 +41,7 @@
 DEFAULT_LOG_FILE = "log.log"
 IS_DEV_MODE = os.environ.get("DEV_MODE", "0") == "1"
 CONFIG_OVERRIDE = [
-    ("fast_sync = true", "fast_sync = false"),
-    ("max_num_outbound_peers = 10", "max_num_outbound_peers = 0"),
+    ("max-num-outbound-peers = 10", "max-num-outbound-peers = 0"),
```

Comment:
- The configuration file field [fastsync] has been renamed to [blocksync].
- The top level configuration file field fast-sync has moved under the new [blocksync] field as blocksync.enable
 (https://github.com/tendermint/tendermint/blob/master/UPGRADING.md#v035)

Reply:
So where is this configuration then here?

Reply:
[upgrade](https://github.com/tendermint/tendermint/blob/master/UPGRADING.md#config-changes-1) lists the following

- The configuration file field [fastsync] has been renamed to [blocksync].
- The top level configuration file field fast-sync has moved under the new [blocksync] field as blocksync.enable.
- blocksync.version = "v1" and blocksync.version = "v2" (previously fastsync) are no longer supported. Please use v0 instead. During the v0.35 release cycle, v0 was determined to suit the existing needs and the cost of maintaining the v1 and v2 modules was determined to be greater than necessary.

and the config.toml looks as follows:
```
#######################################################
###       Block Sync Configuration Connections       ###
#######################################################
[blocksync]

# If this node is many blocks behind the tip of the chain, BlockSync
# allows them to catchup quickly by downloading blocks in parallel
# and verifying their commits
enable = true

# Block Sync version to use:
#   1) "v0" (default) - the standard Block Sync implementation
#   2) "v2" - DEPRECATED, please use v0
version = "v0"
```

however, we need to find a better way to overwrite because there is also statesync which also uses the same key

```
#######################################################
###         State Sync Configuration Options        ###
#######################################################
[statesync]
# State sync rapidly bootstraps a new node by discovering, fetching, and restoring a state machine
# snapshot from peers instead of fetching and replaying historical blocks. Requires some peers in
# the network to take and serve state machine snapshots. State sync is not attempted if the node
# has any local state (LastBlockHeight > 0). The node will have a truncated block history,
# starting from the height of the snapshot.
enable = false
```